lib/animas/animasSimulator.js

```diff
@@ -74,6 +74,21 @@ exports.make = function(config){
     return e;
   }
 
+  function validateDuration(basal) {
+    var fiveDays = (5 * 1440 * sundial.MIN_TO_MSEC);
+    if(basal.isAssigned('duration')) {
+      if(basal.duration > fiveDays) {
+        //flat-rate basal
+        basal.duration = fiveDays;
+        annotate.annotateEvent(basal, 'animas/basal/flat-rate');
```

Comment:
Whether a basal rate is 0 U/hr or 2 U/hr, if it runs for a long time it's a flat rate, right? If you feel strongly that flat rate has a specific connotation in the diabetes world and will be misleading, I'm happy to change it. Keep in mind we'll have to make the corresponding change to tideline too, which in turn means we will be having more discussions around changing the other annotations, e.g., whether we should change the word "capture" to something that better fits Tidepool's voice... ;)